drivers/hid/hid-ids.h

```diff
@@ -385,6 +385,8 @@
 #define USB_DEVICE_ID_DWAV_EGALAX_MULTITOUCH_73F7	0x73f7
 #define USB_DEVICE_ID_DWAV_EGALAX_MULTITOUCH_A001	0xa001
 
+#define I2C_PRODUCT_ID_ELAN_TOUCHPAD 0x303e
```

Comment:
Hmm... I'm not sure you need to put this into header file.
And the name should start with `I2C_DEVICE_ID_` of course.

Reply:
For hid devices we have the convention to gather all vendor / product id defines in this single header.

We also have the convention to simply prefix all IDS with USB_DEVICE_ID_ even if it is an i2c device, the only exception to this is when the vendor uses a different VENDOR_ID for usb vs i2c, which they usually do not.

Reply:
> For hid devices we have the convention to gather all vendor / product id defines in this single header.
> 
> We also have the convention to simply prefix all IDS with USB_DEVICE_ID_ even if it is an i2c device, the only exception to this is when the vendor uses a different VENDOR_ID for usb vs i2c, which they usually do not.

Then I will rename it to USB_DEVICE_ID